### ISSUE TITLE
[Bug]: victoriapark_wa_gov_au source fails on date formatting

### ISSUE THREAD

**Issue by hendo83** [mampfes/hacs_waste_collection_schedule#2684]
### I Have A Problem With:

A specific source

### What's Your Problem

If the bulk waste or green waste start and finish dates span across two months the script fails when adding the address. For example when the dates from the website are like this: "31 Aug-1 Sep 2024, 3-4 May 2025 (Residential only)" 

### Source (if relevant)

victoriapark_wa_gov_au

### Logs

```Shell
The source returned an invalid response: "time data '31 Aug Sep 2024' does not match format '%d %b %Y'". Please check the provided arguments and try again.
```

### Relevant Configuration

```YAML
58 Star Street, Carlisle (example address)
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by dt215git** [comment]
![image](https://github.com/user-attachments/assets/97c2bb9f-16ed-42ca-a990-afc161c127a3)

This looks to be caused by dates that span months - the script seem to assume they'd always be within the same month so the logic constructs a malformed date, like "31 Aug Sep 2024".

It's possible a similar failure mode would occur is dates spanned the year end, but is it likely there would  be waste collections on New Years Eve and New Years Day and the site would be displaying something like "31 Dec 2024-1 Jan 2025"?

**Comment by 5ila5** [comment]
fixed in #2689 

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA

install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
